docs/UPGRADING.md

```diff
@@ -0,0 +1,38 @@
+# Upgrade guides
+
+This document includes the list of required changes when upgrading an existing JSS app.
+Each version contains the list of files, dependencies and environment variables that need to be applied or updated. Unreleased changes are listed in [JSS Next](#jss-next).
+
+Changes are grouped by templates and add-ons.
+
+## JSS Next
+
+### nextjs
```

Comment:
Do we need these now? if so, can we rename it like the headers in the upgrade doc(i.e. [Update SXA add-on files](https://doc.sitecore.com/xmc/en/developers/jss/216/jss-xmc/upgrade-jss-21-5-next-js-apps-to-version-21-6.html#:~:text=nextjs/utils%27%3B%20statement.-,Update%20SXA%20add%2Don%20files,-To%20upgrade%20the)?

Reply:
for work-in-progress entries we do need the per-tempalte sections, makes it easier to add and sort entries, less chace a change is lost or mislabeled. They can be cleaned up when doing release activities.